### ISSUE TITLE
U2F / 2FA minor issue on Chrome

### ISSUE THREAD

**Issue by stefanomarty** [go-gitea/gitea#7085]
- Gitea version (or commit ref): **1.8.2**
- Git version:
- Operating system: **Debian 8.11**
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] **MySQL**
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] **Not relevant**
- Log gist:

## Description
I'm still seeing a minor issue on U2F / 2FA after upgrading to v.1.8.2. I'm using a Yubikey Neo on a Macbook pro.

Login with Firefox works fine:
- I confirm username and password
- When asked, I put the Yubikey in the USB port and press the Yubikey button
- I get in

The same sequence does not work with Chrome (version 74.0.3729.169):
- I confirm username and password
- When asked, I put the Yubikey in the USB port and press the button once
- Nothing happens
- If I do nothing else, I get a "timeout" message (see below)
- If I keep pressing quickly the Y multiple times, sometimes I get in, sometimes I still get the timeout message

On the other hand, If the Yubikey is inserted **before** confirming username and password, it works fine on Chrome as well.

I don't know it this behaviour depends on Chrome or Gitea, I just noticed this difference between Chrome and Firefox.

...

## Screenshots
<img width="720" alt="Schermata 2019-05-30 alle 09 51 45" src="https://user-images.githubusercontent.com/3297621/58619847-ff9a9600-82c5-11e9-9694-26e88b86a8de.png">

**Comment by e3b0c442** [comment]
Curious if you see similar behavior in other U2F sites? Facebook, Google, Gitlab, Github, etc?

**Comment by e3b0c442** [comment]
In attempting to duplicate this in a clean Gitea instance, I am finding that regardless of browser, I am receiving the error when trying to add a U2F key:
  
```
Could not read your security key.
Your browser does not support U2F security keys. 
```

This is also on a MacBook Pro running Mojave 10.14.5. Going to try another client OS and see if the result is the same, then dive further.

**Comment by e3b0c442** [comment]
I am not able to duplicate this with a clean 1.8.2 install using the Docker image, using the latest stable Chrome (74.0.3729.169) on macOS 10.14.5, and a Yubikey 4 firmware version 4.3.7. Sequence of events:

* Log in with username/password
* Gitea prompts to insert security key
* Insert Yubikey
* Yubikey starts blinking
* Touch Yubikey
* Login proceeds successfully

Unfortunately I don't have a NEO nor any other Yubikey version to test against. Yubikey also behaves as expected using Firefox.

**Comment by stefanomarty** [comment]
My mistake, after making several more tests, I found out I was not respecting the initialisation sequence on the Yubikey. BTW, my Yubikey is a 4, not a Neo as I wrote previously.

I still do not understand why I didn't get the same error on Firefox, maybe it was only a coincidence on touching the key at the wrong time.

Turns out that I was pressing the Y on the Yubikey during, or right after, the initial blinking sequence, without waiting for the steady blinking starting a couple of seconds later. That explains why I didn't get any error if the Yubikey was inserted before confirming the username+pwd, as the initialising sequence was already completed when I got the confirming page.

Gitea works fine, error was on my side, sorry for opening an issue which was not an issue at all.
